### DISCUSSION TITLE
Thermostat choice impact on functionality

### DISCUSSION THREAD

**Discussion by JanPannier** [emsesp/EMS-ESP32#1133] [Q&A]
Hello,

I will be installing a new Buderus boiler shortly, and need to select which thermostat to accompany it. 

In how I plan it, the thermostat would only be responsible for measuring temperature, and controlling boiler modulation to meet setpoint temperature. Everything else (heating schedule, hot water control, etc) would be controlled through home assistant. 

My idea was to take the simplest thermostat (e.g. Buderus RC100). Would I lose any options in the gateway/home assistant if I do this vs choosing a more advanced wall thermostat?

**Comment by proddy** [comment]
even an RC20 would do that (you can have mine!). @bbqkees what do you think?
